atspi-common/src/events/mod.rs

```diff
@@ -417,8 +428,8 @@ impl_try_from_event_for_user_facing_type!(
 	CacheEvents::LegacyAdd,
 	Event::Cache
 );
-event_test_cases!(LegacyAddAccessibleEvent);
-impl_from_dbus_message!(LegacyAddAccessibleEvent);
+event_test_cases!(LegacyAddAccessibleEvent, false);
```

Comment:
We spoke about replacing the bool by a more descriptive enum.

Reply:
Now using `Auto/Explicit`